README.md

```diff
@@ -14,37 +14,29 @@ If you have more issues with submodules, please check out [the wiki troubleshoot
 
 ## Getting started / Running the app
 
-Make sure you have node [`v16.4.2`](.nvmrc) installed before installing the dependencies.
-
-If you don't have it, the easiest way to manage node versions is by using nvm. Read the [`Install NVM guide`](https://github.com/nvm-sh/nvm#installing-and-updating).
-
-After installing it, you need to install the correct node version:
+Install dependencies
 
 ```bash
-nvm install 16.4.2
+yarn
 ```
 
-After the installation, just use it:
+Serve with hot reload at localhost:3000
 
 ```bash
-nvm use
+yarn dev
 ```
 
-Now you're ready to install the dependencies and run the app.
+Build for production and launch server
 
 ```bash
-# install dependencies
-$ yarn
-
-# serve with hot reload at localhost:3000
-$ yarn dev
+yarn build
+yarn start
+```
 
-# build for production and launch server
-$ yarn build
-$ yarn start
+Generate static project
 
-# generate static project
```

Comment:
the `#` was lost here. Generate is not part of the running process.

Reply:
Removed from the getting started section.